core/server/master/src/main/java/alluxio/master/MasterProcess.java

```diff
@@ -102,6 +87,8 @@ public MasterProcess(JournalSystem journalSystem, PrimarySelector leaderSelector
     mLeaderSelector = Preconditions.checkNotNull(leaderSelector, "leaderSelector");
     mRpcBindAddress = configureAddress(rpcService);
     mWebBindAddress = configureAddress(webService);
+    mRpcConnectAddress = NetworkAddressUtils.getConnectAddress(rpcService, Configuration.global());
```

Comment:
what's the difference between connect address vs bind address?

Reply:
The bind address is `<ip_address>:<port>` whereas the connect address is `<hostname>:<port>`.